회칙.md

```
@@ -2,73 +2,83 @@
-> 개정: 컴퓨터공학부 강명석  
-> 최신개정일 25-07-27  
+> 최초 작성일: 2025-08-17
+>
+> 최신 개정일: 2025-08-22
+>
+> 최신 개정자: [오현우CSE]()
+>
+> 작성자: [강명석CSE](mailto:tomskang@naver.com), [오현우CSE]()
 
-### 제1 장 총칙
+### 제1장 총칙
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)